modules/oaiharvest/lib/oai_harvest_daemon.py

```diff
@@ -572,36 +572,47 @@ def remove_duplicates(harvested_file_list):
     """
     Go through a list of harvested files and remove any duplicate records.
     """
+    from copy import deepcopy
+    import lxml.etree as etree
```

Comment:
This makes hard requirement on lxml which was only recommended, not mandatory.  Especially on maint-1.1 instances this could create problems.  Can you "fail gracefully" for people that do not have lxml installed?